application/comit_node/tests/rfc003_bitcoin_htlc.rs

```diff
@@ -164,30 +174,25 @@ fn redeem_refund_htlc() {
             htlc.unlock_after_timeout(keypair),
         )],
         output_address: alice_addr.clone(),
-        locktime: 0,
     }
     .sign_with_fee(fee);
 
     let redeem_tx_hex = serialize_hex(&redeem_tx).unwrap();
-
     let raw_redeem_tx = rpc::SerializedRawTransaction(redeem_tx_hex);
 
-    let rpc_redeem_txid_error = client.send_raw_transaction(raw_redeem_tx.clone()).unwrap();
-
-    // It should fail because it's too early
-    assert!(rpc_redeem_txid_error.is_err());
```

Comment:
❓Why is this assert removed?

Reply:
This error is a bit unreliable because depending on the size of the relative timelock the transaction may not be included in the next block even if it was sent successfully.